subsys/esb/esb.c

```diff
@@ -1373,6 +1438,11 @@ int esb_init(const struct esb_config *config)
 
 	disable_event.event.generic.event = esb_ppi_radio_disabled_get();
 
+	if (esb_cfg.optimize_txen_delay) {
+		nrf_radio_modecnf0_set(NRF_RADIO, true, nrf_radio_modecnf0_dtx_get(NRF_RADIO));
```

Comment:
Out of curiosity. Do we need fast ramp-up only for Tx? What about Rx?

Reply:
You are right, this works for both transmitter and receiver.
With your approval, can I rename the option `use_fast_rampup` ?

Reply:
Done.

Reply:
Do you mean something like: 
`int esb_init(const struct esb_config *config, bool use_fast_rampup) {...}`

Reply:
No Just to call `nrf_radio_modecnf0_set` in esb_init and there decide if fast ram-up is used

Reply:
I don't understand what you mean.
Only final user of the SDK can decide if he needs the fast ramp-up, ad it depends on the device that will transmit or receive (only compatible with nRF52/nRF53).

Now this is the code in `esb_init ` function:
```
	if (esb_cfg.use_fast_ramp_up) {
		nrf_radio_modecnf0_set(NRF_RADIO, true, nrf_radio_modecnf0_dtx_get(NRF_RADIO));
	}
```

Do you mean you would prefer (which is actually a lot cleaner and that I will implement anyway ...) 
(nb: the code looks like that because it was originally direct access to registers):
```
	nrf_radio_modecnf0_set(NRF_RADIO, esb_cfg.use_fast_ramp_up, nrf_radio_modecnf0_dtx_get(NRF_RADIO));
```

Reply:
@KAGA164  ?